lock-api/src/main/metrics/metric-schema.yml

```diff
@@ -0,0 +1,13 @@
+options:
+  javaPackage: 'com.palantir.atlasdb.timelock.api'
+
+namespaces:
+  conjureTimelockServiceBlocking:
+    docs: Metrics helpful for timeLock adjudication.
+    metrics:
+      leaderTime:
+        type: timer
+        docs: p99 and rate for leaderTime endpoint on timeLock
```

Comment:
The fact that this logs a p99 and rate is encoded in the information that we have a timer. I think it's enough to say "how long it takes to get leaderTime from timelock" or something